### PR TITLE
Remove code after project migration

### PR DESCRIPTION
**Pull request by gligneul**
The only remaining file is the README

### CODE REVIEW

**Review by guidanoli — COMMENTED** [review]
Repository and branch names could be styled with backticks (like `this`).
Also, I personally think "available in" sounds better in the context it is used.

File: README.md [hunk 1]
```diff
@@ -1,276 +1,9 @@
 # Cartesi Rollups
 
-This repository contains the on-chain and off-chain pieces that are used to deploy, launch and interact with Cartesi Rollups DApps. The code presented here is work in progress, continuously being improved and updated.
+The Cartesi Rollups project migrated to new repositories.
+The on-chain code is available in the [`cartesi/rollups-contracts` repository](https://github.com/cartesi/rollups-contracts).
+The off-chain code is available in the [`cartesi/rollups-node` repository](https://github.com/cartesi/rollups-node).
```
Comment (guidanoli) [inline README.md:5]: The README looks nicer in the HTML page if we skip every line here.

**Review by guidanoli — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (guidanoli) [reply README.md]: Yes, all good.

**Review by tuler — DISMISSED** [review]

**Review by guidanoli — APPROVED** [review]

**Comment by tuler** [comment]
I just found a link at the documentation straight to https://github.com/cartesi/rollups#portals
This is the kind of link that will break.

**Comment by gligneul** [comment]
@tuler the documentation should point to the specific tag; in this case:  https://github.com/cartesi/rollups/tree/v1.0.0#portals
@miltonjonat @claudioantonio ^

**Comment by guidanoli** [comment]
Totally agree with @gligneul on this. We should never assume things won't change in the future. Using tags and commit is the way to be sure these links won't break or point to the wrong piece of information.

**Pull request merged**